app/assets/javascripts/base.js.erb

```diff
@@ -6,6 +6,8 @@
 //= require honeybadger-js/dist/honeybadger.js
 //= require serviceworker-companion
 //= require algolia/v3/algoliasearch.min
+//= require jquery
+//= require jquery_ujs
```

Comment:
I did this to make the `remote: true` flag function on the `button_to` helper. I'm curious why we weren't doing this, and subsequently if I shouldn't do this.. :sweat_smile:

Reply:
Though I don't remember what's the alternative to `remote: true` IIRC both were intentionally left out from the bundle as we don't want to include jquery and jquery_ujs in all pages. I think jQuery it's only required in some pages, see `app/views/pages/rlyweb.html.erb`, `app/views/pages/generator.html.erb`, `app/views/videos/new.html.erb` and others

cc @benhalpern

Reply:
Hmm I'm not sure what our alternative to `remote: true` is either. For previous stuff I've used the browser `fetch` and put it in a new Webpacker JS file.

Reply:
Yeah, we left out ujs because it more than doubles our bundle size.